tools/SourceKit/lib/SwiftLang/SwiftEditor.cpp

```diff
@@ -2170,7 +2170,7 @@ void SwiftLangSupport::editorOpen(
 
   if (!Snapshot) {
     Snapshot = EditorDoc->initializeText(
-        Buf, Args, Consumer.needsSemanticInfo(), fileSystem);
+        Buf, Args, Consumer.needsSemanticInfo());
```

Comment:
This means that re-opening a document without closing it first will silently re-use the original filesystem. I'm fine with that, since we don't really support this anyway, but it's worth calling out.

Reply:
aha, there is a test exercising this and it caught this change. I will change it back to the original behavior. (My client is relying on this reopening behavior right now, though I do want to eventually change it not to, since you say it's not really supported.)